app/experimenter/experiments/tests/test_migrations.py

```diff
@@ -1,48 +1,99 @@
-from django.db import connection
-from django.db.migrations.executor import MigrationExecutor
-from django.test.testcases import TransactionTestCase
+from django.utils.text import slugify
+from django_test_migrations.contrib.unittest_case import MigratorTestCase
```

Comment:
The template we had didn't work for this?

Reply:
No, I fell down a rabbit hole of needing to access models from the pre-migration and post-migration states because of the change in fields. Things broke down when I tried to use the existing factories and the User model. This `django-test-migrations` package wrapped it all up pretty concisely

Reply:
Ah yep makes sense 👍